stl/inc/utility

```diff
@@ -490,18 +489,19 @@ struct tuple_size<tuple<_Types...>> : integral_constant<size_t, sizeof...(_Types
 };
 
 template <size_t _Index>
-struct tuple_element<_Index, tuple<>> { // enforce bounds checking
+struct _MSVC_KNOWN_SEMANTICS tuple_element<_Index, tuple<>> { // enforce bounds checking
     static_assert(_Always_false<integral_constant<size_t, _Index>>, "tuple index out of bounds");
 };
 
 template <class _This, class... _Rest>
-struct tuple_element<0, tuple<_This, _Rest...>> { // select first element
+struct _MSVC_KNOWN_SEMANTICS tuple_element<0, tuple<_This, _Rest...>> { // select first element
     using type   = _This;

```

Comment:
As Casey noted, the addition of the comment affects clang-format's "align consecutive assignments". Please remember to make an identical change to your Microsoft-internal MSVC-PR-232587.